scripts/mattermost-mysql-6.0.0.sql

```diff
@@ -1177,15 +1177,15 @@ CREATE TABLE `Users` (
 /*!40101 SET character_set_client = @saved_cs_client */;
 
 --
--- Table structure for table `schema_migrations`
+-- Table structure for table `db_migrations`
 --
 
 /*!40101 SET @saved_cs_client     = @@character_set_client */;
 /*!40101 SET character_set_client = utf8 */;
-CREATE TABLE `schema_migrations` (
-  `version` bigint(20) NOT NULL,
-  `dirty` tinyint(1) NOT NULL,
-  PRIMARY KEY (`version`)
+CREATE TABLE `db_migrations` (
```

Comment:
Is it correct to remove the old table? A 6.0 DB will already have `schema_migrations` table. Maybe we want to add the new table instead or change the schema of this table itself?

Reply:
The structure of two tables are different, and also how they interpreted. I think we shouldn't try to use existing one. If the app is going to start with a fresh db it's not necessary to create a table for go-migrate.